Find a functionally equivalent code:android.graphics.drawable.GradientDrawable.setGradientCenter:COMMENT
Method Modifier: public      
Comment:/**
 * <p>Sets the center location of the gradient. The radius is honored only when
 * the gradient type is set to {@link #RADIAL_GRADIENT} or {@link #SWEEP_GRADIENT}.</p>
 * <p><strong>Note</strong>: changing this property will affect all instances
 * of a drawable loaded from a resource. It is recommended to invoke
 * {@link #mutate()} before changing this property.</p>
 *
 * @param x The x coordinate of the gradient's center
 * @param y The y coordinate of the gradient's center
 *
 * @see #mutate()
 * @see #setGradientType(int)
 */

@@ -1,5 +1,5 @@
 {
     mGradientState.setGradientCenter(x, y);
-    mRectIsDirty = true;
+    mGradientIsDirty = true;
     invalidateSelf();
 }

